### PR TITLE
About page

### PR DESCRIPTION
**Pull request by chasulin**
Includes all content in `cms-content-entry` branch (all new about page content, learn updates)

### CODE REVIEW

File: src/screens/About/Team/Team.style.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { Paragraph } from 'components/Markdown';
+import Grid from '@material-ui/core/Grid';
+import { COLOR_MAP } from 'common/colors';
+
+export const Headshot = styled.img`
+  border-radius: 50%;
+  max-width: 4rem;
+  max-height: 4rem;
```
Comment (ghop02) [inline src/screens/About/Team/Team.style.tsx:9]: css question, should rem units be used for widths over pixels?

Comment (ghop02) [inline src/screens/About/Team/Team.style.tsx:9]: Would setting this as width fix the case when the images aren't properly sized? We should probably just find a way to make sure all the images are the same size

File: src/screens/About/Team/TeamSection.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        const alphabetizedTeam = sortBy(
+          team.teamMembers,
+          member => member.fullName,
+        );
+        const teamToMap =
+          team.teamName === 'Leadership' ? team.teamMembers : alphabetizedTeam;
+        const isAlumni = team.teamName === 'Alumni';
+        const includeTitle = teamsWithTitles.includes(team.teamName);
+        return (
+          <Fragment>
```
Comment (ghop02) [inline src/screens/About/Team/TeamSection.tsx:22]: Does this need to be a fragment over a div or grid container?

File: src/screens/About/Team/TeamSection.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+const TeamSection = () => {
+  return (
+    <Fragment>
+      {teams.map(team => {
+        const alphabetizedTeam = sortBy(
+          team.teamMembers,
+          member => member.fullName,
+        );
+        const teamToMap =
```
Comment (ghop02) [inline src/screens/About/Team/TeamSection.tsx:17]: nit: I think `teamMembers` might be more clear here - took me a sec to realize that what "to map" was for

File: src/screens/About/About.style.tsx [hunk 4]
```diff
@@ -1,81 +1,31 @@
...
-import { TeamList } from 'screens/About/About';
-import { COLOR_MAP } from 'common/colors';
-import ReactMarkdown from 'react-markdown';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+import { Heading1, Heading2, Heading3 } from 'components/Markdown';
 
-export const Wrapper = styled.div`
-  background-color: white;
-  min-height: calc(100vh - 64px);
+export const AboutHeading1 = styled(Heading1)`
```
Comment (ghop02) [inline src/screens/About/About.style.tsx:5]: Will we eventually be streamlining these to lessen the need for custom headings?

**Review by ghop02 — APPROVED** [review]
minor nits about some styling but looks good to me!

File: src/cms-content/team/people-operations.json [hunk 5]
```diff
@@ -1,29 +1,29 @@
...
+      "profileImgUrl": "/images_cms/shaeleighjacobs.jpeg",
       "fullName": "Shaeleigh Jacobs",
       "description": "People operations lead",
       "profileUrl": "https://www.linkedin.com/in/shaeleigh-jacobs-672152114/"
     },
     {
-      "profileImgUrl": "/images_cms/bar_recommends_icon.svg",
+      "profileImgUrl": "/images_cms/brittany.jpeg",
       "fullName": "Brittany Braxton",
       "profileUrl": "Brittany Braxton"
```
Comment (ghop02) [inline src/cms-content/team/people-operations.json:12]: oh i noticed that Brittany's profile url isn't actually a link

File: src/screens/About/Team/TeamSection.tsx (hunk 2 above)
Comment (chasulin) [reply src/screens/About/Team/TeamSection.tsx:22]: this fragment contains the grid container (which contains a grid item, in this case `AlumniMember` or `ActiveMember`). react doesnt let you return adjacent elements (in this case `AboutHeading3` and a `Grid` container) without wrapping them in a parent element--the fragment acts as the parent without adding anything to the dom.

File: src/screens/About/About.style.tsx (hunk 4 above)
Comment (chasulin) [reply src/screens/About/About.style.tsx:5]: thats the plan. this page needed more spacing than our heading styles have built into them :(

File: src/screens/About/GovLogoGrid.tsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { GovLogo } from 'components/LogoGrid/LogoGrid.style';
+
+const GovLogoGrid = (props: { logos: any }) => {
```
[the remark this answers is not in the document]
Comment (chasulin) [reply src/screens/About/GovLogoGrid.tsx:5]: added the types!

File: src/screens/About/Team/Team.style.tsx (hunk 1 above)
Comment (chasulin) [reply src/screens/About/Team/Team.style.tsx:9]: hmm my very incomplete understanding of the use of rems vs. pixels is that rems are better for accessibility. but tbd

**Pull request merged**